### ISSUE TITLE
Don't support `??` operator?

### ISSUE THREAD

**Issue by taosimple** [ethanblake4/dart_eval#102]
```
    num main () {
      var myfunc = ([num? a, num? b]) {
        return (a ?? 0) + (b ?? 0);
      };
      return myfunc(2, 4);
    }
```
evla error:
```
package:dart_eval/src/eval/compiler/compiler.dart 394:7   Compiler.compileSources
package:dart_eval/src/eval/compiler/compiler.dart 142:12  Compiler.compile
package:dart_eval/src/eval/compiler/compiler.dart 444:21  Compiler.compileWriteAndLoad
test/function_test.dart 197:32                            main.<fn>.<fn>

CompileError: Unknown binary operator QUESTION_QUESTION at unknown (file package:example/main.dart)
```

**Comment by ethanblake4** [comment]
Support added in v0.6.1

**Comment by ethanblake4** [comment]
Reopening as this does not work correctly in v0.6.1

**Comment by ethanblake4** [comment]
Fixed in v0.6.2
